Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -1461,6 +1470,58 @@ def installRectilinearWipePipeline(self):
 
         self.AddActor(wipeSlice, WIPE_ACTOR)
 
+    def installSliceSliderWidgetPipeline(self):
```

Comment:
Please add docstrings to the methods you modified/added in this file. I know there are not many at the moment but as the code grows it becomes harder for the reviewers to follow